docs/tutorials/local-da-devnet.md

```diff
@@ -0,0 +1,323 @@
+---
+sidebar_position: 3
+sidebar_label: GN World - Local DA Tutorial
+description: Run your Rollkit rollup while connected to a local Celestia Data Availability network.
+---
+
+# GN World
+
+:::caution Note
+This tutorial has only been tested on an AMD machine running Ubuntu 22.10 x64.
+:::
+
+This tutorial will teach developers how to easily run a local data availability (DA) devnet on their own machine (or in the cloud).
+Running a local devnet for DA to test your rollup is the recommended first step before deploying to a testnet.
+This eliminates the need for testnet tokens and deploying to a testnet until you are ready.
+
+The development journey for your rollup will look something like this:
+
+1. Run your rollup and post DA to a local devnet, and make sure everything works as expected
+2. Deploy the rollup, posting to a DA testnet. Confirm again that everything is functioning properly
+3. Finally, deploy your rollup to the DA Layer's mainnet
+
+Whether you're a developer simply testing things on your laptop or using a virtual machine in the cloud,
+this process can be done on any machine of your choosing. We tested it out on a machine with the following specs:
+
+- Memory: 1 GB RAM
+- CPU: Single Core AMD
+- Disk: 25 GB SSD Storage
+- OS: Ubuntu 22.10 x64
+
+## 🌕 Running Local Devnet with a Rollkit Rollup
+
+First, run the [`local-celestia-devnet`](https://github.com/celestiaorg/local-celestia-devnet) by running the following command:
```

Comment:
Missing a prerequisites section to install Docker

Reply:
awesome, thanks yaz

Reply:
31f5f0c324cb775b85522a51647f5ba65fc67350